src/Microsoft.AspNetCore.Http/HttpContextFactory.cs

```diff
@@ -43,18 +43,12 @@ public HttpContext Create(IFeatureCollection featureCollection)
                 throw new ArgumentNullException(nameof(featureCollection));
             }
 
-            var responseCookiesFeature = new ResponseCookiesFeature(featureCollection, _builderPool);
-            featureCollection.Set<IResponseCookiesFeature>(responseCookiesFeature);
```

Comment:
This change breaks feature chaining. See https://github.com/aspnet/Security/blob/6294badd972882ba707216c211095474ef8c4709/src/Microsoft.AspNetCore.CookiePolicy/CookiePolicyMiddleware.cs#L29